src/sentry/workflow_engine/endpoints/validators/base/action.py

```diff
@@ -66,13 +66,17 @@ def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
 
         attrs = super().validate(attrs)
 
+        if not Action.Type(attrs["type"]).is_integration() and attrs.get("integration_id"):
```

Comment:
Is an `integration_id` required for integrations? should this also be checking to see if it's an integration and doesn't have the integration id?

if we need to check multiple, might be nice to give these booleans names -- if not, feel free to ignore this nit
```py
is_integration = Action.Type(attrs["type"]).is_integration()
has_integration_id = attrs.get("integration_id") is not None

if not is_integration and has_integration:
   raise ...
   ```

Reply:
Hmm good point. Then I can remove the extra check in the handlers